src/maintenance/azext_maintenance/tests/latest/test_maintenance_scenario.py

```diff
@@ -323,7 +324,7 @@ def step__maintenanceconfigurations_delete_maintenanceconfigurations_deleteforre
 def step__maintenanceconfigurations_delete_publicmaintenanceconfigurations_delete(test):
     test.cmd('az maintenance configuration delete '
              '--resource-group "{rg}" '
-             '--resource-name "sqlcli" '
+             '--resource-name "{publicMaintenanceConfiguration}" '
```

Comment:
Please use fake value as value, don't use placeholder such as {xxx}, this expression will confuse users.

Reply:
The resource name is a placeholder value, but the same resource is used in multiple tests (create, get, and delete tests). If the resource names differ for each test, the tests would fail, resulting in a "Resource not found" error. To avoid repeating the same hardcoded value across those tests, I added the resource name in kwargs. I believe this will also improve the readability of the tests.

Please let me know if you still think that we should repeat the hardcoded values across the interrelated tests.